subprojects/packagefiles/libarchive/cat/meson.build

```diff
@@ -0,0 +1,11 @@
+bsdcat = executable(
+  'bsdcat',
+  files(
+    'bsdcat.c',
+    'cmdline.c',
+  ),
```

Comment:
Since we're discussing moving over to files() I think it's worth noting that *inlining* the files() function as an argument to executable() is totally pointless. It's also less readable.

Reply:
(Now granted, `muon fmt` doesn't discern that order isn't significant here. But it *could*.)

Reply:
> Since we're discussing moving over to files() I think it's worth noting that _inlining_ the files() function as an argument to executable() is totally pointless. It's also less readable.

I actually find the separation between the name of the executable and the sources more readable, but would just use an array:
```python
bsdcat = executable(
  'bsdcat',
  [
    'bsdcat.c',
    'cmdline.c',
  ],
  dependencies: libarchive_fe_dep,
  install: true,
)
```

Reply:
Sure, ultimately the important thing is a) self-consistency, followed by b) what the people primarily contributing to that wrap find easy to work with.

With that in mind, whichever one you choose should be consistent between here and `libarchive_fe`. :)

Reply:
> Sure, ultimately the important thing is a) self-consistency, followed by b) what the people primarily contributing to that wrap find easy to work with.

You forgot: potential compatibility with dumb tools (`meson rewrite`…) :P.

> With that in mind, whichever one you choose should be consistent between here and `libarchive_fe`. :)

Dammit!